ast.go

```diff
@@ -0,0 +1,93 @@
+package sshconfigo
+
+// String is a string inside a line. It does not contain new line endings.
+type String struct {
+	// Pos is the position of the first character in the line.
+	Pos int
+	// Text is the contents of the string.
+	Text string
+}
+
+// Line is a line inside a ssh_config file. An empty line is represented by a
+// Line that does not contain a value for the keyword.
+type Line struct {
+	// Pos is the line number.
+	Pos int
+	// Prefix contains the spaces and tabs before the keyword.
+	Prefix String
+	// LiteralKeyword contains the keyword as it is found in the file. It's case
+	// insensitive.
+	LiteralKeyword String
+	// Assignment is the separator between the keyword and the arguments. It is
+	// usually a space, it may contain the '=' character.
+	Assignment String
+	// LiteralArguments contains the arguments as they are found in the file.
+	// It's case sensitive.
+	LiteralArguments String
+	// Postfix contains the spaces and tabs after the arguments.
+	Postfix String
+	// Comment is the optional string starting the '#' character.
+	Comment String
+	// NewLine contains the new line endings as they are found in the file.
+	NewLine String
+}
+
+// IsBlank returns true when the Line does not contain a value for the keyword,
+// that is when it is empty or there are only spaces and tabs and / or there is
+// only a comment.
+func (l Line) IsBlank() bool {
+	return l.Keyword() == ""
+}
+
+// Keyword returns the canonical name of the LiteralKeyword if exists, or the
+// LiteralKeyword otherwise.
+func (l Line) Keyword() string {
+	// TODO Implement Keyword
+	return ""
+}
+
+// Arguments returns the arguments after the escape characters have been
+// processed.
+func (l Line) Arguments() string {
+	// TODO Implement Arguments
+	return ""
+}
+
+// ArgumentsAsSlice returns the arguments after the escape characters have been
+// processed.
+func (l Line) ArgumentsAsSlice() []string {
+	// TODO Implement ArgumentsAsSlice
+	return nil
+}
+
+// Section represents a slice of lines starting from line with the Host
+// specification.
+type Section struct {
+	// Host is the Line with the Host specification.
+	Host Line
+	// Config is the slice of Lines belonging to the Section. It contains all the
+	// Lines until the next Section or end of file.
+	Config []Line
```

Comment:
Not sure whether `Config` would be the best name for this. It would be a bit generic, what do you think?

Reply:
Suggest a better one. Lines? SubLines? Properties?

Reply:
I would say `Lines` is more appropriate since it may contain blank lines.